src/Websites/Websites/help/Publish-AzWebApp.md

```diff
@@ -15,15 +15,17 @@ Deploys an Azure Web App from a ZIP, JAR, or WAR file using zipdeploy.
 ### FromWebApp (Default)
 ```
 Publish-AzWebApp -ArchivePath <String> [-Type <String>] [-Clean] [-Async] [-Restart] [-TargetPath <String>]
- [-IgnoreStack] [-Reset] [-Force] [-AsJob] [-Timeout <Double>] [-WebApp] <PSSite>
- [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+ [-IgnoreStack] [-Reset] [-UseZipDeploy] [-Force] [-AsJob] [-Timeout <Double>] [-WebApp] <PSSite>
+ [-DefaultProfile <IAzureContextContainer>] [-ProgressAction <ActionPreference>] [-WhatIf] [-Confirm]
```

Comment:
I think ProgressAction is removed on "main" branch. Was this added back intentionally or just need to rebase your branch on main.